src/cli.rs

```diff
@@ -0,0 +1,93 @@
+use structopt::StructOpt;
+
+use notion_core::session::Session;
+use notion_fail::Fallible;
+
+use crate::command::{self, Command};
+
+#[derive(StructOpt)]
+#[structopt(
+    name = "Notion",
+    about = "The hassle-free JavaScript toolchain manager",
+    author = "",
+    usage = "notion [flags] [<command> <args> ...]
+
+    To install a tool in your user toolchain, use `notion install`.
+    To pin a tool in a project toolchain, use `notion pin`.",
```

Comment:
Can you speak to the rationale behind making this part of the general help message, instead of being part of the `Use` command? It seems a little odd to have a few specific subcommands called out in the `USAGE` block, while everything is shown in the `SUBCOMMANDS` block.

Reply:
Ah, I see: I misunderstood what was going on there! Here and for your question below: we can totally have a hidden subcommand that displays this for backwards compatibility. I'll push a commit to that effect shortly!